lib/mappings/createNew.js

```diff
@@ -1,7 +1,38 @@
-import { createWallet } from '../actions';
+import assert from 'bsert';
+import { selectWallets, selectCurrentChain, selectNetwork } from './selectors';
+import { createWallet, createAccount } from '../actions';
+import { getxpub } from '../utilities';
+
+/*
+ * getxpub then create account using the xpub
+ */
+
```

Comment:
Maybe this function name is too specific? `getxpub` only works with hardware for now, but it technically doesn't have to

Reply:
Yeah, it seems like this would make sense as `createWatchOnlyAcct`, whereas the `getxpub` is actually specific to hardware wallets right?